solidity/ecdsa/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@keep-network/ecdsa",
+  "version": "2.0.0-dev",
+  "description": "Keep ECDSA Wallet",
+  "license": "MIT",
+  "scripts": {
+    "format": "npm run lint",
+    "format:fix": "npm run lint:fix",
+    "lint": "npm run lint:eslint && npm run lint:sol && npm run lint:config",
+    "lint:fix": "npm run lint:fix:eslint && npm run lint:fix:sol && npm run lint:config:fix",
+    "lint:eslint": "eslint .",
+    "lint:fix:eslint": "eslint . --fix",
+    "lint:sol": "solhint 'contracts/**/*.sol' && prettier --check '**/*.sol'",
+    "lint:fix:sol": "solhint 'contracts/**/*.sol' --fix && prettier --write '**/*.sol'",
+    "lint:config": "prettier --check '**/*.@(json|yaml)'",
+    "lint:config:fix": "prettier --write '**/*.@(json|yaml)'",
+    "clean": "hardhat clean",
+    "build": "hardhat compile",
+    "test": "hardhat test",
+    "deploy": "hardhat deploy --export export.json",
+    "prepack": "tsc -p tsconfig.json && hardhat export-artifacts export/artifacts"
+  },
+  "devDependencies": {
+    "@keep-network/hardhat-helpers": "^0.4.1-pre.2",
+    "@keep-network/hardhat-local-networks-config": "^0.1.0-pre.4",
+    "@nomiclabs/hardhat-ethers": "^2.0.3",
+    "@nomiclabs/hardhat-waffle": "^2.0.1",
+    "@tenderly/hardhat-tenderly": "^1.0.13",
+    "@thesis-co/eslint-config": "github:thesis/eslint-config",
+    "@thesis/prettier-config": "github:thesis/prettier-config",
+    "@typechain/ethers-v5": "^8.0.5",
+    "@typechain/hardhat": "^3.1.0",
+    "@types/chai": "^4.3.0",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^17.0.5",
+    "chai": "^4.3.4",
+    "eslint": "^7.32.0",
+    "eslint-config-keep": "github:keep-network/eslint-config-keep",
```

Comment:
I think we agreed on using thesis-co wide linter, which is added in line 29. We don't have eslint-config-keep in /random-beacon

Reply:
a2a61cdd552fd010d6ee26c5dd0ccaa0a5ddae85